core/src/main/java/io/questdb/griffin/engine/groupby/GroupByUtils.java

```diff
@@ -260,6 +260,15 @@ public static void toTop(ObjList<? extends Function> args) {
         }
     }
 
+    public static void clear(ObjList<? extends GroupByFunction> args) {
```

Comment:
It would be better if `clear()` was alongside `Misc.close()` in `Misc` class and clear() should also take this signature:

```
ObjList<? extends Mutable>
```

Reply:
Makes sense. Done in 8059ba7c32c1a1ac7ac2f3bdd3d66b16fadf624b